website/_includes/nav_footer_custom.html

```diff
@@ -0,0 +1 @@
+<!-- This file removes the just-the-docs reference in the nav bar's footer. -->
```

Comment:
It's what we did on the website branch, I believe because we thought it to be cleaner at the time. I am fine with removing it to give some credits, if others agree.

Reply:
Can't recall adding this, but might have just forgotten. Let's add it back.

Reply:
Added back.